cpg_workflows/jobs/count.py

```diff
@@ -37,33 +38,38 @@ class FeatureCounts:
     """
 
     def __init__(
-            self,
-            input_bam: BamPath | str | Path,
-            gtf_file: str | Path,
-            output_path: str | Path,
-            summary_path: str | Path,
-            paired_end: bool = True,
-            feature_type: str = 'exon',
-            attribute: str = 'gene_id',
-            strandness: str = 'reverse',  # one of: 'none', 'forward', 'reverse'
-            multi_mapping: bool = False,
-            min_quality: int | None = None,
-            primary_only: bool = False,
-            ignore_duplicates: bool = False,
-            count_pairs: bool = True,
-            both_ends_mapped: bool = True,
-            both_ends_same_chr: bool = True,
-            threads: int = 1,
-        ) -> None:
+        self,
+        input_bam: BamPath | str | Path,
+        gtf_file: str | Path,
+        output_path: str | Path,
+        summary_path: str | Path,
+        paired_end: bool = True,
+        feature_type: str = 'exon',
+        attribute: str = 'gene_id',
+        strandness: str = 'reverse',  # one of: 'none', 'forward', 'reverse'
+        multi_mapping: bool = False,
+        min_quality: int | None = None,
+        primary_only: bool = False,
+        ignore_duplicates: bool = False,
+        count_pairs: bool = True,
+        both_ends_mapped: bool = True,
+        both_ends_same_chr: bool = True,
+        threads: int = 1,
+    ) -> None:
         self.command = [
             'featureCounts',
-            '-t', feature_type,
-            '-g', attribute,
-            '-s', {'none': '0', 'forward': '1', 'reverse': '2'}[strandness],
-            '-a', str(gtf_file),
-            '-T', str(threads),
+            '-t',
+            feature_type,
+            '-g',
+            attribute,
+            '-s',
+            {'none': '0', 'forward': '1', 'reverse': '2'}[strandness],
+            '-a',
+            str(gtf_file),
+            '-T',
+            str(threads),
```

Comment:
Gah! Another example of "this semantically-blind autoformatting is not making this easier to read"